src/services/conversationsService.js

```diff
@@ -88,6 +88,33 @@ const checkTalkVersionHash = function(response) {
 	store.dispatch('setNextcloudTalkHash', newTalkCacheBusterHash)
 }
 
+/**
+ * Fetch listed conversations
+ * @param {string} searchTerm The string that will be used in the search query.
+ */
+const searchListedConversations = async function(searchTerm) {
+	try {
+		const response = await axios.get(generateOcsUrl('apps/spreed/api/v3', 2) + 'listed-room', {
+			params: {
+				searchTerm,
+			},
+		})
+
+		if (maintenanceWarning) {
+			maintenanceWarning.hideToast()
+			maintenanceWarning = null
+		}
+
+		return response
+	} catch (error) {
+		if (error.response && error.response.status === 503 && !maintenanceWarning) {
+			maintenanceWarning = showError(t('spreed', 'Nextcloud is in maintenance mode, please reload the page'), {
```

Comment:
This should be handled globally. Feels weird that this search would trigger it but posting a message would not?

Reply:
so something for later?

Reply:
I copy-pasted this from another method. Agreed that this should be global. I'll remove it